rust/macros/module.rs

```diff
@@ -604,7 +628,10 @@ pub(crate) fn module(ts: TokenStream) -> TokenStream {
                 }}
             }}
 
-            fn __exit() {{
+            /// # Safety
+            /// This function should only be called by the kernel to ensure exclusive access to the local `static mut __MOD` variable.
+            unsafe fn __exit() {{
+                // SAFETY: This function will only be called by the kernel when the module gets unloaded which ensures, that we have exclusive access.
```

Comment:
Nit: this line is a bit too long (134).

Reply:
Done, tried to shorten every comment to 100 characters per line